plugins/SegmentEditor/lang/en.json

```diff
@@ -20,10 +20,12 @@
         "SegmentIsDisplayedForWebsite": "and displayed for",
         "SegmentNotApplied": "Segment '%s' not applied",
         "SegmentNotAppliedMessage": "You are requesting data for the Custom Segment '%s', this Piwik configuration currently prevents real time processing of reports for performance reasons.",
-        "SelectSegmentOfVisitors": "Select a segment of visitors:",
+        "SelectSegmentOfVisitors": "Select a segment of visits:",
```

Comment:
Maybe we should also change the key in that case, as it would differ from the translation otherwise